src/ExpenseTrackerGrupo2/Persistence/Database/Scripts/init.sql

```diff
@@ -0,0 +1,77 @@
+CREATE TABLE IF NOT EXISTS Users (
+    Id SERIAL PRIMARY KEY,
+    Name VARCHAR(100) NOT NULL,
+    Email VARCHAR(100) NOT NULL UNIQUE,
+    PasswordHash VARCHAR(255) NOT NULL,
+    CreatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS Expenses (
+    Id SERIAL PRIMARY KEY,
+    UserId INT NOT NULL,
+    Amount DECIMAL(10, 2) NOT NULL,
+    Description VARCHAR(255),
+    Source VARCHAR(100),
+    Month INT NOT NULL,
+    GoalAmount DECIMAL(10, 2),
+    Category VARCHAR(100),
+    Date DATE NOT NULL,
+    CreatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY (UserId) REFERENCES Users(Id)
```

Comment:
Have you considered the delete type given the relationship we have between user and other entities?

Reply:
i dont unterdstand the comment if you can explain more

Reply:
Okay! Here is a link with that.
https://www.postgresql.org/docs/current/ddl-constraints.html

You can go directly to this part and below
![image](https://github.com/user-attachments/assets/967a0744-b100-4238-8aac-412f39bdb28e)

Reply:
Okay! Here is a link with that.
https://www.postgresql.org/docs/current/ddl-constraints.html

You can go directly to this part and below
![image](https://github.com/user-attachments/assets/967a0744-b100-4238-8aac-412f39bdb28e)